layouts/partials/head/meta.html

```diff
@@ -5,7 +5,14 @@
 <!-- End of Google Analytics -->
 
 <!-- Default meta.html content -->
-<meta name="Description" content="{{ .Site.Params.description }}">
+<meta name="description" content="{{ .Site.Params.description }}">
+<meta property="og:type" content='{{ if eq .Section "posts" }}article{{ else }}website{{ end }}'>
+<meta property="og:url" content="{{ .Permalink }}">
+<meta property="og:site_name" content="{{ .Site.Title }}">
+<meta property="og:title" content="{{ .Title }}">
+<meta property="og:description" content="{{ with .Summary }}{{ . }}{{ else }}{{ .Site.Params.detailedDescription }}{{ end }}">
+<meta property="og:image" content="{{ .Site.Params.home.profile.avatarURL}}">
```

Comment:
```suggestion
<meta property="og:image" content="https://prepas-mp2i.fr{{ .Site.Params.home.profile.avatarURL}}">
```
Les url relatives ne marchent pas : https://stackoverflow.com/questions/70049918/fb-opengraph-ogimage-link-a-folder